assignment3.txt

```diff
@@ -0,0 +1,15 @@
+/*Rebecca Hale, Part 2 Assignment 2*/
+1.
+a. The complexity is n.
+b. The complexity is n^2 + n.
```

Comment:
Close! since it's big O notation, you don't have to care about the lower terms, so this just becomes O(n^2).